packages/amplify-appsync-simulator/API.md

```diff
@@ -268,6 +270,8 @@ export type AppSyncSimulatorSchemaConfig = AppSyncMockFile;
 export type AppSyncSimulatorServerConfig = {
     port?: number;
     wsPort?: number;
+    sslKeyPath?: string;
+    sslCertPath?: string;
```

Comment:
It is not obvious from this interface that providing ssl key and cert enables HTTPS.

Could you please rework this shape to something like this ?

```
export type AppSyncSimulatorServerConfig = {
   httpsConfig?: {
     sslKeyPath: string;
     sslCertPath: string;
   }
}

```